airgun/entities/host.py

```diff
@@ -62,6 +62,14 @@ def change_host_environment(self, entities_list, values):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def read_yaml_output(self, entity_name):
+        """Get puppet external nodes YAML dump for specific host"""
+        view = self.navigate_to(self, 'Details', entity_name=entity_name)
+        view.yaml_dump.click()
+        output = view.browser.element(view.yaml_output).text
```

Comment:
why not `view.yaml_output.text`?

Reply:
okay, logically they are the same, but in fact they did different actions for me. I am not sure that I want to debug that as it is not big deal
`view.browser.element(view.yaml_output).text` returned string with all `\n` symbols, but `view.browser.text(view.yaml_output)` returned without

Reply:
nvm, let's me try to test `view.yaml_output.text`

Reply:
tried `output = view.yaml_output.text`. Also doesn't work for me and crash entire run